AAP-3275 added release notes for 2.1.2 and 2.1.3

emurtoug opened the pull request:
Updated [2.1 Release notes](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/red_hat_ansible_automation_platform_release_notes/index) with notes for 2.1.2 and 2.1.3.

ShaiahWren commented on release-notes/topics/aap-212.adoc:
```diff
@@ -0,0 +1,56 @@
+* Added support for `ansible_host` in the installer inventory for automation hub
+* Fixed an issue where a few commands were missing on initializing the container
+* Fixed an issue where the TTL controller caused the jobs to run repeatedly
+* Resolved intermittent `500 internal server error` while pulling the execution environments from private automation hub
```
Our changelog (https://github.com/ansible/galaxy_ng/pull/1178/files) for 2.1.2/4.4.3 also mentions fixing the 500 server error when pushing EE's. Should this also be mentioned here?

emurtoug replied:
I modified the language of this note to reflect this.

ShaiahWren commented on release-notes/topics/aap-212.adoc:
```diff
@@ -0,0 +1,56 @@
+* Resolved issues when upgrading automation hub from version 2.1.1 to version 2.1.2
+* Resolved issues where the Controller user interface was not accessible, when automation controller was installed through Operator on OpenShift Core Platform 4.10
+* Fixed an issue that caused the upgrade from Ansible Automation Platform 1.2 to 2.1 to fail, when the execution environment was transferred to automation hub
+* Fixed an issue so that a standalone installation of private automation hub can now transfer the execution environments to the hub
```
Does this correspond to https://issues.redhat.com/browse/AAH-1373 ticket?

emurtoug replied:
I believe this corresponds to https://issues.redhat.com/browse/AAP-2112. I have added another note to reference https://issues.redhat.com/browse/AAH-1373 below.

ShaiahWren reviewed (commented):
Release 2.1.2 (4.4.3) notes looks copacetic with our changelogs in [galaxy](https://github.com/ansible/galaxy_ng/pull/1178/files) and [auto-hub](https://github.com/ansible/ansible-hub-ui/pull/1776/files), with the exception of a couple tickets that may be missing: https://issues.redhat.com/browse/AAH-1384 , https://issues.redhat.com/browse/AAH-1411, and https://issues.redhat.com/browse/AAH-1389. They may  be worded differently in the docs so just let me know if they are already accounted for.

ShaiahWren commented on release-notes/topics/aap-212.adoc:
```diff
@@ -0,0 +1,58 @@
+* Updated the proxy password from text type to password type to make the password more secure
+* Updated the synchronization that is used by authenticated proxy to avoid authentication failures
+* Added message in the Controller user interface to inform users that the community collections do not have docs
+* Resolved intermittent issue while trying to sync community `devsec.hardening` collection on automation hub 4.4.0
```
Does this correspond to https://issues.redhat.com/browse/AAH-1202? If so, it should appear in the 2.1.2 (4.4.4) release notes, according to our [changelog](https://github.com/ansible/galaxy_ng/pull/1386/files)

ShaiahWren commented on release-notes/topics/aap-213.adoc:
```diff
@@ -0,0 +1,34 @@
+* While using automation hub 4.5.0 with central authentication, the hub user interface no longer turns unresponsive when the group permissions are modified
+* Added localization support for *Modules*, *Roles*, *Plugins*, and *Dependencies* counter in container list for both card and list view
+* Updated the documentation link to route to documentation of the correct version
+* Resolved intermittent issue while trying to sync community `devsec.hardening` collection on automation hub 4.4.0
```
Ah, it looks like https://issues.redhat.com/browse/AAH-1202 may have been accidentally included in the 2.1.1 notes above, since they are redundant :)

emurtoug replied:
https://issues.redhat.com/browse/AAH-1202 has been removed from 2.1.2.

ShaiahWren commented:
Items I'm unsure are/should be included in notes (see [galaxy_ng changelog](https://github.com/ansible/galaxy_ng/pull/1386/files):
https://issues.redhat.com/browse/AAH-1349, https://issues.redhat.com/browse/AAH-1557, https://issues.redhat.com/browse/AAH-1749, https://issues.redhat.com/browse/AAH-1737

and in our [auto-hub changelog](https://github.com/ansible/ansible-hub-ui/pull/2377/files):
https://issues.redhat.com/browse/AAH-1228,
https://issues.redhat.com/browse/AAH-1573,
https://issues.redhat.com/browse/AAH-1527

emurtoug commented:
> Items I'm unsure are/should be included in notes (see [galaxy_ng changelog](https://github.com/ansible/galaxy_ng/pull/1386/files)): https://issues.redhat.com/browse/AAH-1349, https://issues.redhat.com/browse/AAH-1557, https://issues.redhat.com/browse/AAH-1749, https://issues.redhat.com/browse/AAH-1737
> 
> and in our [auto-hub changelog](https://github.com/ansible/ansible-hub-ui/pull/2377/files): https://issues.redhat.com/browse/AAH-1228, https://issues.redhat.com/browse/AAH-1573, https://issues.redhat.com/browse/AAH-1527

Hi @ShaiahWren, 

Thanks again for your thorough review! I have added the following notes: 

**Within 2.1.1:** 
https://issues.redhat.com/browse/AAH-1228 
Relese note text: Updated ansible-hub-ui to use a changelog

**Within 2.1.2: N/A**

**Within 2.1.3:**

- https://issues.redhat.com/browse/AAH-1527 
Release note text: Added visibility for execution environments created by ansible-builder in the automation hub UI
- https://issues.redhat.com/browse/AAH-1349
Resolved issue while copying and removing content from a repository by changing lock and transaction into a single task
- https://issues.redhat.com/browse/AAH-1749
Release note text: Updated orphan cleanup task to improve collection re-upload experience
- https://issues.redhat.com/browse/AAH-1737
Release note text: Resolved an issue where a list of execution environments available and discovered were not displayed after indexing
- https://issues.redhat.com/browse/AAH-1573
Release note text: Updated user access to be able to customize minimum password length

I did not include https://issues.redhat.com/browse/AAH-1557, because this Jira notes that it is not to be included within the release notes. 

Can you check that you're happy with the placement of these notes and with the language used to describe them? 

Thanks again!

ShaiahWren commented:
> * Added visibility for execution environments created by ansible-builder in the automation hub UI

For some reason I'm not seeing this in the release notes yet - am I missing something?

ShaiahWren commented:
> https://issues.redhat.com/browse/AAH-1349
Resolved issue while copying and removing content from a repository by changing lock and transaction into a single task

Same as above

emurtoug commented:
> > https://issues.redhat.com/browse/AAH-1349
> > Resolved issue while copying and removing content from a repository by changing lock and transaction into a single task
> 
> Same as above

Hi @ShaiahWren!

Thanks again for your review. Please see lines 35 and 36 of release-notes/topics/aap-213.adoc for these updates.

ShaiahWren commented:
LGTM

ianf77 reviewed (approved):
LGTM. One observation on release notes in general, which you can ignore.
